docs/sampling.asciidoc

```diff
@@ -1,27 +1,77 @@
 This means that only `20%` of traces will send and retain all of their associated information.
 The remaining traces will drop contextual information to reduce the transfer and storage size of the trace.
 
+Head-based sampling is quick and easy to set up.
+Its downside is that it's entirely random -- good
+data might be discarded purely due to chance.
+
+See <<configure-head-based-sampling>> to get started.
+
+**Distributed tracing with head-based sampling**
+
+In a distributed trace, the sampling decision is still made when the trace is initiated.
+Each subsequent service respects the initial service's sampling decision, regardless of its configured sample rate;
+the result is a sampling percentage that matches the initiating service.
+
+In this example, `Service A` initiates four transactions and has sample rate of `.5` (`50%`).
+The sample rates of `Service B` and `Service C` are ignored.
+
+image::./images/dt-sampling-example-1.png[Distributed tracing and head based sampling example one]
+
+In this example, `Service A` initiates four transactions and has a sample rate of `1` (`100%`).
+Again, the sample rates of `Service B` and `Service C` are ignored.
+
+image::./images/dt-sampling-example-2.png[Distributed tracing and head based sampling example two]
+
 [float]
-==== Why sample?
+[[tail-based-sampling]]
+==== Tail-based sampling
 
-Distributed tracing can generate a substantial amount of data,
-and storage can be a concern for users running `100%` sampling -- especially as they scale.
+In tail-based sampling, the sampling decision for each trace is made after the trace has completed.
+This means all traces will be analyzed against a set of rules, or policies, which will determine the rate at which they are sampled.
 
-The goal of probability sampling is to provide you with a representative set of data that allows
-you to make statistical inferences about the entire group of data.
-In other words, in most cases, you can still find anomalous patterns in your applications, detect outages, track errors,
-and lower MTTR, even when sampling at less than `100%`.
+Tail-based sampling reduces the risk of discarding important data, because the sampling decision is only made _after_
+each trace has been analyzed.
+However, because traces are all initially observed,
+storage and transfer costs may be higher than with head-based sampling.
+
+See <<configure-tail-based-sampling>> to get started.
+
+**Distributed tracing with tail-based sampling**
+
+With tail-based sampling, all traces are observed and a sampling decision is only made once a trace completes.
+
+In this example, `Service A` initiates four transactions.
+If our sample rate is `.5` (`50%`) for traces with a `success` outcome,
+and `1` (`100%`) for traces with a `failure` outcome,
+the sampled traces would look something like this:
+
+image::./images/dt-sampling-example-3.png[Distributed tracing and tail based sampling example one]
 
 [float]
-==== What data is sampled?
+=== Sampled data
```

Comment:
As above, we should probably update these docs, as we no longer store non-sampled trace events as of 8.0. (Again, RUM is special.)

Reply:
Yeah, errors are exempt from sampling. It's just transactions and spans that may be dropped.